appveyor.yml

```diff
@@ -4,9 +4,16 @@ image: Visual Studio 2015
 environment:
   QT5DIR: C:\Qt\5.8
 
+  matrix:
+      - PYTHON: "C:\Python35"
+        PYTHON_VERSION: "3.5.x"
+        PYTHON_ARCH: "32"
```

Comment:
So `PYTHON` variable can be just moved straight to `environment:`, like `QT5DIR`.